bolt/src/main/java/com/slack/api/bolt/AppConfig.java

```diff
@@ -67,8 +69,12 @@ private EnvVariableName() {
         public static final String SLACK_APP_OAUTH_COMPLETION_URL = "SLACK_APP_OAUTH_COMPLETION_URL";
     }
 
+    // We don't use SlackConfig.DEFAULT here to enable developers to customize the properties later on
     @Builder.Default
-    private transient Slack slack = Slack.getInstance(SlackConfig.DEFAULT, buildSlackHttpClient());
+    private transient Slack slack = Slack.getInstance(new SlackConfig(), buildSlackHttpClient());
```

Comment:
As we support the way to load `AppConfig` from a JSON file data, this object must be transient (= excluded from the properties that should be loaded from JSON string)